compiler/damlc/daml-doc/src/DA/Daml/Doc/Transform/Annotations.hs

```diff
@@ -23,7 +27,7 @@ applyMove = map (foldr1 g) . groupSortOn (modulePriorityKey . md_name) . map f
         g m1 m2 = ModuleDoc
             { md_anchor = md_anchor m1
             , md_name = md_name m1
-            , md_descr = md_descr m1
+            , md_descr = md_descr m1 <|> md_descr m2
```

Comment:
This looks like for anything but `Prelude` we will somewhat randomly (it’s deterministic but unclear which order) chose which module description we use. Should we maybe change this to always use the one from the module that we are moving to and simply ignore the description in all modules that are being moved?

Reply:
`md_descr` for moved modules is set to `Nothing`, so this is the behavior we now have. Before we were dropping module descriptions somewhat randomly...

Reply:
Makes sense, thanks for the explanation! Might be nice to document this in a comment.